sdk/src/client/secret/ledger_nano.rs

```diff
@@ -166,6 +169,16 @@ impl SecretManage for LedgerSecretManager {
         Ok(ed25519_addresses)
     }
 
+    async fn generate_evm_addresses(
+        &self,
+        _coin_type: u32,
+        _account_index: u32,
+        _address_indexes: Range<u32>,
+        _options: Option<GenerateAddressOptions>,
+    ) -> Result<Vec<EvmAddress>, Self::Error> {
+        Err(Error::UnsupportedOperation)
```

Comment:
hm nano cant do this?

Reply:
Thoralf said so

Reply:
either way, lets get this merged and see if someone needs it. I thought this: https://developers.ledger.com/docs/embedded-app/crypto-examples/#11a-key-generation menas it can

Reply:
@kwek20 it can, but not the iota/shimmer apps, needs to be the ethereum app and this one we don't support here